simple/stats/cache.py

```diff
@@ -27,13 +30,39 @@
 from stats.data import Triple
 from stats.db import Db
 
+STAT_VAR_GROUPS_CACHE_KEY = "StatVarGroups"
```

Comment:
what does this key mean?  Does it need to match something in the mixer?  Please comment.

Reply:
This is the key used to store the SVG hierarchy in the `key_value_store` table. [Examples](https://github.com/search?type=code&q=org%3Adatacommonsorg+statvargroups+language%3Acsv+repo%3Adatacommonsorg%2Fimport)

Yes, [mixer](https://github.com/datacommonsorg/mixer/blob/fae6f51bdce56b669b00a9e5d79314a4dd6087b2/internal/sqldb/sqlquery/key_value.go#L27) uses the same key.